mantis-publish-web-sample/buildDockerImage.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# build the mantis-server-worker fat jar
+../gradlew clean build
+
+# build the Docker image that packages the mantis-server-worker along with a running mesos-slave
+docker build -t dev/mantispublishweb .
```

Comment:
Same for this one.

Reply:
Yes plan to do that in the next iteration. Wanted to get the sample out first.